libs/accounts/src/lib/asset-data-provider.ts

```diff
@@ -0,0 +1,24 @@
+import { makeDataProvider } from '@vegaprotocol/react-helpers';
+import { AssetsDocument } from './__generated__/Accounts';
+import type {
+  AssetsQuery,
+  AssetsFieldsFragment,
+} from './__generated__/Accounts';
+
+const getData = (responseData: AssetsQuery) =>
+  responseData.assetsConnection?.edges?.reduce((aggr, edge) => {
```

Comment:
suggestion (non blocking): filter/map might be more readable here? There isn't really any perf reasons yet to prevent the additional loops
```
responseData.assetsConnection.edges
  .filter(e => Boolean(e.node))
  .map(e => e.node)
```